recipes/pymatnest/build.sh

```diff
@@ -0,0 +1,9 @@
+make libs
+cp *.py ${SP_DIR}
+cp *.so ${SP_DIR}
+cp ns_analyse ${PREFIX}/bin
+cp ns_run ${PREFIX}/bin
+cp ns_profile_analyse ${PREFIX}/bin
+cp ns_process_traj ${PREFIX}/bin
+cp ns_T_dependent_analysis ${PREFIX}/bin
+cp ns_set_analysis_weights ${PREFIX}/bin
```

Comment:
Copying straight to the site packages dir is pretty dirty. Why is this done?

Reply:
Nested sampling is a rather theoretical approach - rarely used in practise. Still it is nice for educational purposes. The code is not nice, but it is the best implementation I am aware off. 

**Background:** 
As part of the https://github.com/pyiron project I started to submit a series of materials science codes to conda-forge - the full list is available here https://jan-janssen.github.io/conda-forge-contribution/ . We agreed early on to have the packages in the conda-forge channel, rather than building a separate channel like bioconda.

Reply:
There are lots of great nested sampling packages. See multinest or polychord.

Reply:
Sorry, I meant nested sampling for solid state materials science. But I agree, most likely it is more reasonable to use one of the mainstream codes and build a materials science specific version on top of it. Still at the moment the pymatnest code is the one used in our specific case. If you say that it does not match the conda-forge quality level that is fine for me, then I move it to a separate channel.

Reply:
it's not quality. I just want to make sure this is actually what the upstream authors intend. The vast major of people won't install it anyway